### PR TITLE
update to POMDPs.jl v0.9

### PR DESCRIPTION
**Pull request by rcnlee**
Update ContinuumWorld.jl to be compatible with new interface in POMDPs.jl v0.9, also fix some warnings from Plots.jl.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -9,7 +9,7 @@ using Random
 
 w = CWorld()
 
-@inferred gen(DDNOut(:sp, :r), w, [3,2], [4,2], MersenneTwister(19))
+@gen(:sp, :r)(w, [3,2], [4,2], MersenneTwister(19))
```
Comment (zsunberg) [inline test/runtests.jl]: I think we can still use `@inferred` here. Does it fail the test?

File: src/ContinuumWorld.jl [hunk 2]
```diff
@@ -52,11 +54,11 @@ const default_rewards = [-10.0, -5.0, 10.0, 3.0]
...
 POMDPs.actions(w::CWorld) = w.actions
-POMDPs.n_actions(w::CWorld) = length(w.actions)
 POMDPs.discount(w::CWorld) = w.discount
 
-function POMDPs.gen(::DDNNode{:sp}, w::CWorld, s::AbstractVector, a::AbstractVector, rng::AbstractRNG)
-    return s + a + w.stdev*randn(rng, Vec2)
+function POMDPs.gen(w::CWorld, s::AbstractVector, a::AbstractVector, rng::AbstractRNG)
+    sp = s + a + w.stdev*randn(rng, Vec2)
+    return (sp=sp,)
 end
```
Comment (zsunberg) [inline src/ContinuumWorld.jl]: When implementing only the state transition, it is better to use `POMDPModelTools.ImplicitDistribution` than `gen`. `gen` is for when you need to return multiple things (reward, observation, state) from one function.

```suggestion
function POMDPs.transition(w::CWorld, s::AbstractVector, a::AbstractVector)
    ImplicitDistribution(w, s, a) do w, s, a, rng
        return s + a + w.stdev*randn(rng, Vec2)
    end
end
```

**Review by zsunberg — CHANGES_REQUESTED** [review]
Hey Ritchie, thanks so much for doing this! I made a few suggestions. I'll also send you an email with my thoughts about this type of problem (don't want to share everything publicly because it involves the solution to a HW problem in my class, haha)

File: test/runtests.jl (hunk 1 above)
Comment (rcnlee) [reply test/runtests.jl]: Yes, you're right, we should keep the ``@inferred`` here.  I just tested it and it passes.  Can you suggest the edit and I'll commit it? Thanks!

Comment (zsunberg) [reply test/runtests.jl]: Haha, I don't think I need to suggest it - there should be a way for you to just make the change, even on the github interface.

Comment (zsunberg) [reply test/runtests.jl]: In any case, I think it should be
```suggestion
@inferred @gen(:sp, :r)(w, [3,2], [4,2], MersenneTwister(19))
```

Comment (rcnlee) [reply test/runtests.jl]: Thanks! I can't seem to find the github button to suggest the change on my side...

**Comment by mykelk** [comment]
@zsunberg should this be merged?

**Comment by zsunberg** [comment]
Yes, I think so.

**Comment by zsunberg** [comment]
@mykelk is someone working on better continuous space support. If so, I have a lot of thoughts. I kinda had to give up on working on that after the summer, but it could be done

**Comment by mykelk** [comment]
I'll ping the SISL slack to see who is interested in contributing!

**Comment by mykelk** [comment]
@zsunberg : @jamgochiana is interested in contributing! We can coordinate by email.

**Pull request merged**